tools/cardano-sl-tools.cabal

```diff
@@ -261,8 +243,7 @@ executable cardano-swagger
 
   if flag(for-installer)
     buildable: False
-
-  if !flag(with-wallet) || !flag(with-web)
+  else
     buildable: False
```

Comment:
This is also wrong, we need to be able to build keygen. You can either move it to `cardano-sl-wallet` or make `cardano-sl-tools` depend on `cardano-sl-wallet` (which is fine).